config/hubs/pangeo-hubs.cluster.yaml

```diff
@@ -24,14 +24,12 @@ support:
       controller:
         admissionWebhooks:
           enabled: false
-    nfs-server-provisioner:
-      enabled: false
```

Comment:
I guess this is not related at all, just removing old stuff??

Reply:
Correct, it's disabled by default here: https://github.com/2i2c-org/pilot-hubs/blob/85a9adb707aef3bbcb8a95947960114a860a8761/support/values.yaml#L75-L76